File: Data Structures & Algorithms/concatenation-of-array/submission-0.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        int len = nums.size();
        vector<int> result(len * 2);
        for(int i = 0; i < len; i++){
            int num = nums[i];
            result[i] = num;
            result[len + i] = num;
        }
        return result;
    }
};

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @param {number} k
     * @return {number[]}
     */
    topKFrequent(nums, k) {
        const map = new Map();
        for (let num of nums){
            if(!map.has(num)){
                map.set(num, 0)
            }
            map.set(num, map.get(num) + 1)
        }
        const keys = new Map([...map].sort((a, b) => b[1] - a[1])).keys() 
        return Array.from(keys).slice(0,k)
    }
}
